src/Adapter/Driver/PdoDriverInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Db\Adapter\Driver;
+
+interface PdoDriverInterface extends DriverInterface {}
```

Comment:
Should this be refactored out into the base package, so that all satellites can use it?

Reply:
It's in the base package. If it were not then it would be in the adapter-mysql repo.